src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1173,28 +1173,25 @@ private Symbol UnwrapAlias(Symbol symbol, out AliasSymbol alias, BindingDiagnost
             }
             else
             {
+                var boundTypeArguments = BindTypeArguments(typeArguments, diagnostics, basesBeingResolved);
+                if (unconstructedType.IsGenericType
+                    && options.IsAttributeTypeLookup()
+                    && boundTypeArguments.FirstOrDefault(bta => bta.Type.IsTypeParameter() || bta.Type.IsUnboundGenericType() || bta.Type.ContainsTypeParameter()) is { HasType: true } badAttributeArgument)
```

Comment:
>bta.Type.IsTypeParameter() [](http://example.com/codeflow?start=64&length=26)

Can this case be removed and handled by `ContainsTypeParameter()`?

Reply:
Yes.